specification/search/data-plane/Azure.Search/indexreadme.md

```diff
@@ -0,0 +1,480 @@
+# SearchIndexClient
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for SearchIndexClient.
+
+
+---
+## Getting Started
+
+To build the SDK for SearchIndexClient, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+### Basic Information
+These are the global settings for SearchIndexClient.
+
+``` yaml
+openapi-type: data-plane
+tag: package-2019-05
+```
+
+
+### Tag: package-2019-05-searchservice-preview
+
+These settings apply only when `--tag=package-2019-05-searchservice-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-05-searchservice-preview'
+input-file:
+- preview/2019-05-06-preview/searchservice.json
+```
+
+### Tag: package-2019-05-searchindex-preview
```

Comment:
Assuming you invoke AutoRest separately for each client, the tags shouldn't need "searchindex" or "searchservice" in them anymore to distinguish them.